src/cockpit/bridge.py

```diff
@@ -210,7 +210,7 @@ def start_ssh_agent() -> None:
     # Launch the agent so that it goes down with us on EOF; PDEATHSIG would be more robust,
     # but it gets cleared on setgid ssh-agent, which some distros still do
     try:
-        proc = subprocess.Popen(['ssh-agent', 'sh', '-ec', 'echo $SSH_AUTH_SOCK; read a'],
+        proc = subprocess.Popen(['ssh-agent', 'sh', '-ec', 'echo SSH_AUTH_SOCK=$SSH_AUTH_SOCK; read a'],
```

Comment:
Why not just change the check below to `.startswith('/')`?  It saves us the parsing...